spec/unit/network/http/handler_spec.rb

```diff
@@ -133,7 +133,7 @@ def respond(text)
 
     it "should raise an error if the request is formatted in an unknown format" do
       handler.stubs(:content_type_header).returns "unknown format"
-      expect { handler.request_format(request) }.to raise_error
+      expect { handler.request_format(request) }.to raise_error(NameError, /undefined local variable or method/)
```

Comment:
"undefined local variable" is surprising, though maybe it's trying to lookup a method named "to_unknown format"?